go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -188,10 +188,24 @@ func (ovn *Controller) createLoadBalancerRejectACL(lb string, sourceIP string, s
 	vip := util.JoinHostPortInt32(sourceIP, sourcePort)
 	// NOTE: doesn't use vip, to avoid having brackets in the name with IPv6
 	aclName := fmt.Sprintf("%s-%s:%d", lb, sourceIP, sourcePort)
+	aclName = strings.ReplaceAll(aclName, ":", "\\:")
+	// ACL names are limited to 63 characters
```

Comment:
maybe add a comment that to solve this limitation you are shortening the name of the load balancer based on the length of the IP+port string.

Reply:
Done